static/js/components/inputs/FieldsOfStudySelectField.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import _ from 'lodash';
+import SelectField from './SelectField';
+import FIELDS_OF_STUDY from '../../fields_of_study';
+
+let fieldOfStudyOptions = _.map(FIELDS_OF_STUDY, (name, code) => ({
+  value: code,
+  label: name
+}));
```

Comment:
![ss 2016-06-15 at 4 39 28 pm](https://cloud.githubusercontent.com/assets/14932219/16096636/db6542f4-3317-11e6-8dbe-dc3d87c3b4bc.png)

moved it to the top. just doesn't want to execute first